build.gradle.kts

```diff
@@ -9,11 +9,18 @@ plugins {
 	id("org.sonarqube") version "2.8"
 }
 
+plugins.withType<JavaPlugin>().configureEach {
+	configure<JavaPluginExtension> {
+		modularity.inferModulePath.set(true)
+	}
+}
+
+
 group = "org.junit-pioneer"
 description = "JUnit 5 Extension Pack"
 
 java {
-	sourceCompatibility = JavaVersion.VERSION_1_8
+	sourceCompatibility = JavaVersion.VERSION_11
```

Comment:
Does the resulting JAR file still run on Java 8?

Reply:
Hmm. I have to reed up if Java 8 can handle the module-info. *look at Nicolais book in my shelf*

Reply:
JPMS came with Java 9. Java 8 works with `module-info.java` by basically ignoring it, afaik.

Reply:
> JPMS came with Java 9. Java 8 works with `module-info.java` by basically ignoring it, afaik.

Not when the module-file is in src/main :D But I got it.